fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1618,7 +1618,7 @@ public class Config extends ConfigBase {
      * The maximum number of partitions to fetch from the metastore in one RPC.
      */
     @ConfField
-    public static int max_hive_partitions_per_rpc = 1000;
```

Comment:
Is this variable used before?

Reply:
Yes. See `CachingHiveMetastore.refreshPartitions`

Reply:
Why increase this value?

Reply:
Empirical value, suggested by @stephen-shelby